blocks/events-form/events-form.js

```diff
@@ -408,7 +408,10 @@ function personalizeForm(form, resp) {
 
   Object.entries(resp).forEach(([key, value]) => {
     const matchedInput = form.querySelector(`#${snakeToCamel(key)}`);
-    if (matchedInput) matchedInput.value = value;
+    if (matchedInput) {
+      matchedInput.value = value;
+      matchedInput.disabled = "true";
```

Comment:
I imagine an actual Boolean value would work here too? Should we use that instead?

Reply:
Unfortunately, the DOM API does not understand boolean. `true` would not work

Reply:
let me check something else. Make sure the cache is not affecting this

Reply:
Seems my local overrides just keep acting up. Anyhow, I pushed the Boolean type. Seems to work.